tests/manual_test_ddex.py

```diff
@@ -20,6 +20,7 @@
 import time
 
 from web3 import Web3, HTTPProvider
+import web3.utils
```

Comment:
this will end up being `_utils` in the updated web3 version right?

Reply:
yep, good catch. I didn't push up the manual test file ive been using locally